modules/ztp-du-host-bios-requirements.adoc

```diff
@@ -0,0 +1,87 @@
+// CNF-1500 ZTP
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="configuring_bios_for_du_hosts_{context}"]
+= Configuring BIOS for distributed unit bare-metal hosts
+
+Distributed unit (DU) hosts require the BIOS to be configured before the host can be provisioned. The BIOS configuration is dependent on the specific hardware that runs your DUs and the particular requirements of your installation.
+
+.Procedure
+. Set the *UEFI/BIOS Boot Mode* to `UEFI` and ensure that the same setting is applied to the `bootMode` field in the `BareMetalHost` custom resource definition (CRD), for example:
++
+[source,yaml]
+----
+provisioning:
+  bootMode: UEFI
+----
+
+. In the host boot sequence order, set *Hard drive first*.
++
+[NOTE]
+====
+If `rootDeviceHints` is specified in the `BareMetalHost` CRD, the hard disk must match the hard disk that you configure in the host boot sequence order, for example:
+
+[source,yaml]
+----
+rootDeviceHints:
+  deviceName: /dev/sda
+----
+====
+
+. Apply the specific BIOS configuration for your hardware. The following table describes a representative BIOS configuration for an Intel Xeon Skylake or Intel Cascade Lake server, based on the Intel FlexRAN 4G and 5G baseband PHY reference design.
++
+[IMPORTANT]
+====
+The exact BIOS configuration will depend on your specific hardware and network requirements. The following sample configuration is for illustrative purposes only.
+====
++
+.Sample BIOS configuration for an Intel Xeon Skylake or Cascade Lake server
+[cols=2*, width="90%", options="header"]
+|====
+|BIOS Setting
+|Configuration
+
+|CPU Power and Performance Policy
+|Performance
+
+|Uncore Frequency Scaling
+|Disabled
+
+|Performance P-limit
+|Disabled
+
+|Enhanced Intel SpeedStep (R) Tech
+|Enabled
+
+|Intel Configurable TDP
+|Enabled
+
+|Configurable TDP Level
+|Level 2
+
+|Intel(R) Turbo Boost Technology
+|Enabled
+
+|Energey Efficient Turbo
+|Disabled
+
+|Hardware P-States
+|Disabled
+
+
+|Package C-State
+|C0/C1 state
+
+|C1E
+|Disabled
+
+|Processor C6
+|Disabled
+|====
+
+[NOTE]
+====
+If you are using the Intel vRAN Dedicated Accelerator ACC100 in your DU, enable global SR-IOV and VT-d settings in the BIOS for the host.
```

Comment:
I don't believe this is only for ACC100 - this should be for all hosts.

Reply:
@juphoff can you comment on this?